change/@fluentui-react-a80addea-99f6-4096-9383-b0b193004086.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31314&commitId=669f19645059369401648f55f4d4cae9edf1e542&env=prod&clientType=FLUENTUI)


  <details><summary><b>DatePicker</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker.Required.hover datepicker.chromium.png|7| Changed|
|DatePicker.Required.default.chromium.png|7| Changed|
|DatePicker.Underlined and Required.hover datepicker.chromium.png|7| Changed|
|DatePicker.Underlined and Required.default.chromium.png|7| Changed|
|DatePicker.Required.hover month.chromium.png|7| Changed|
|DatePicker.Underlined and Required.hover day.chromium.png|7| Changed|
|DatePicker.Required.hover day.chromium.png|7| Changed|
|DatePicker.Required.click.chromium.png|7| Changed|
|DatePicker.Underlined and Required.click.chromium.png|7| Changed|
|DatePicker.Underlined and Required.hover month.chromium.png|7| Changed|

</details>


  <details><summary><b>TextField</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Root.click.chromium.png|2| Changed|
|TextField.Placeholder.click.chromium.png|2| Changed|
|TextField.Error.click.chromium.png|2| Changed|
|TextField.Icon.click.chromium.png|2| Changed|
|TextField.Placeholder - RTL.click.chromium.png|2| Changed|
|TextField.Multiline - RTL.click.chromium.png|2| Changed|
|TextField.Suffix - RTL.click.chromium.png|2| Changed|
|TextField.Multiline.click.chromium.png|2| Changed|
|TextField.Required.click.chromium.png|2| Changed|
|TextField.Error - RTL.click.chromium.png|2| Changed|
|TextField.Icon - RTL.click.chromium.png|2| Changed|
|TextField.Suffix.click.chromium.png|2| Changed|
|TextField.Multiline nonresizable.click.chromium.png|2| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
oh no, looks like the required asterisk renders outside and having overflow hidden hides it 😅